src/System.Net.NetworkInformation/src/System/Net/NetworkInformation/LinuxTcpConnectionInformation.cs

```diff
@@ -0,0 +1,27 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace System.Net.NetworkInformation
+{
+    // This class is not Linux-specific. It is just a data container.
+    // TODO: Combine this with SystemTcpConnection. Make Windows construct that object differently.
```

Comment:
Or just rename the one to be `WindowsTcpConnectionInformation` instead of `System...`